MfciPkg/Application/MfciPolicy/MfciPolicy.py

```diff
@@ -0,0 +1,207 @@
+# @file
+#
+# Write MFCI packet into the MFCI mailbox
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+import os
+import sys
+import logging
+import argparse
+import ctypes
+from UefiVariableSupport.UefiVariablesSupportLib import UefiVariable

```

Comment:
Can you check if this import fails when executing this script from anywhere that is not the workspace root of MU_PLUS? Maybe at the root of a repo when MU_PLUS is a submodule. I'm not confident it will work.

Reply:
I moved the script (and the subfolder) outside of the repo and ran it a couple of different ways and it appears to work (was able to run and return correct information)